src/modules/name/index.ts

```diff
@@ -220,22 +220,36 @@ export class Name {
   /**
    * Return a random gender.
    *
-   * @param binary Whether to return only binary gender names. Defaults to `false`.
+   * @param binary (deprecated) Whether to return only binary gender names. Defaults to `false`.
    *
    * @example
    * faker.name.gender() // 'Trans*Man'
-   * faker.name.gender(true) // 'Female'
    */
   gender(binary?: boolean): string {
     if (binary) {
-      return this.faker.helpers.arrayElement(
-        this.faker.definitions.name.binary_gender
-      );
+      deprecated({
+        deprecated: 'faker.name.gender(true)',
+        proposed: 'faker.name.sex()',
+        since: '7.4',
+        until: '8.0',
+      });
+
+      return this.faker.name.sex();
     }
 
     return this.faker.helpers.arrayElement(this.faker.definitions.name.gender);
   }
 
+  /**
+   * Return a random sex.
```

Comment:
Not sure about the phrasing here:
```suggestion
   * Return a random sex (localized versions of `female`, `intersex`, or `male`).
```
So that it's clear that there are three of them and this method returns them, although I would kind of like to have an optional param to return only a binary sex.

Please also add a `@see` targeting the gender function and vice versa to mentally bind them together.

Reply:
I changed it slightly to express its localized. I can add a line that the output of this function should not be used as a param value whenever we ask for `sex`, but not sure if it's needed...

`@see` added

Reply:
> I can add a line that the output of this function should not be used as a param value whenever we ask for `sex`, but not sure if it's needed...

I think it doesnt hurt to have this. In the worst case we can link to it if someone missused that method.

Reply:
Done